app/views/posts/show.html.erb

```diff
@@ -1 +1,52 @@
-<h1>this is going to show the user post page</h1>
\ No newline at end of file
+<section class="my-bio bg-secondary bg-gradient">
+    <div class="container">
+            <div class="card my-4">
+                <div class="card-header post-heading">
+                    <h4># <%=@post.title%></h4> <h4>Comments: <%=@post.comments_count%> Likes: <%=@post.likes_count%></h4>
```

Comment:
- [ ] Kindly fix this error on the post show page.

- [ ] I could not verify if the page matches [the wireframe](https://github.com/microverseinc/curriculum-rails/raw/main/blog-app/images/blog_single_post.png). Kindly ensure that happens.

  ![image](https://github.com/Izebiafe/solo-blog/assets/81468675/8998a920-7a9d-418f-a0e7-cc7756a35c44)